### PR TITLE
fix: check that full balance becomes available to the client

### PR DESCRIPTION
**Pull request by joschisan**

### CODE REVIEW

File: modules/fedimint-lnv2-tests/src/bin/tests.rs [hunk 1]
```diff
@@ -260,7 +265,15 @@ async fn test_payments(dev_fed: &DevJitFed) -> anyhow::Result<()> {
...
     }
 
-    info!("Testing LDK Gateway can pay HOLD invoice...");
+    while client.balance().await? < 9000 * 1000 {
+        sleep_in_test(
+            format!("Waiting for the full balance to become available to the client"),
+            Duration::from_secs(1),
+        )
+        .await;
+    }
```
Comment (elsirion) [inline modules/fedimint-lnv2-tests/src/bin/tests.rs]: You can use [`retry`](https://docs.fedimint.org/fedimint_core/util/fn.retry.html) here

File: scripts/tests/lnv2-module-test.sh [hunk 2]
```diff
@@ -8,4 +7,4 @@ source scripts/_common.sh
 build_workspace
 add_target_dir_to_path
 
-tests all
```
Comment (elsirion) [inline scripts/tests/lnv2-module-test.sh:11]: Did the `all` ever have any meaning?

Comment (joschisan) [reply scripts/tests/lnv2-module-test.sh:11]: yes. we used to be able to run parts of the test suite seperately but that was removed.

File: modules/fedimint-lnv2-tests/bin/tests.rs [hunk 3]
```diff
@@ -260,7 +264,18 @@ async fn test_payments(dev_fed: &DevJitFed) -> anyhow::Result<()> {
         await_receive_claimed(&client, receive_op).await?;
     }
 
-    info!("Testing LDK Gateway can pay HOLD invoice...");
+    retry(
```
Comment (m1sterc001guy) [inline modules/fedimint-lnv2-tests/bin/tests.rs:267]: Was there a flake that prompted this change? I would think that `await_receive_claimed` would wait until the balance was been updated, so the retry wouldnt be necessary

**Review by elsirion — APPROVED** [review]

**Pull request merged**